examples/start_stop/main.cc

```diff
@@ -0,0 +1,176 @@
+#include <zendar/api/api.h>
+
+#include <gflags/gflags.h>
+
+#include <iostream>
+#include <fstream>
+#include <vector>
+#include <thread>
+#include <chrono>
+
+
+DEFINE_string(
+  device_addr, "localhost",
+  "IPv4 address of the device to connect to. This can be a resolvable hostname "
+  "or given in dotted notation."
+);
+
+DEFINE_string(
+  install_path, "/usr/bin",
+  "Where the Zendar binaries are installed."
+);
+
+DEFINE_string(
+  imaging_mode, "",
+  "The imaging mode to be run."
+);
+
+DEFINE_string(
+  stream_addr, "localhost",
+  "Where the device should stream the image data. "
+  "**Note** This flag tells the device where to send the image data, so this "
+  "address should be from the device's viewpoint."
+);
+
+DEFINE_string(
+  run_duration, "120",
+  "How long (in seconds) to run this program before disconnect. "
+);
```

Comment:
Suggest `DEFINE_int32` instead of `DEFINE_string`

Reply:
Agree.

Reply:
Thank you for catching it.